canonical-web-and-design/canonical.com
# New layout for career game results page

## Done

- New layout for career game results page  
    - Top 4 jobs at the top in cards
    - List of jobs by department below
- Things to fix
    - The departments are still using the old ones like 'cloud engineering' which we hide from the user normally, I noticed it is visible on the live site as well in the department 'message'
    - ![image](https://user-images.githubusercontent.com/441217/111901515-9d79ca80-8a30-11eb-881c-af1dad0abd19.png)
    - These departments links don't lead to real pages
    - The list shows departments witout matching roles?
    - The bottom list doesn't exclude the top 4 roles
    - I don't invite someone to retake the test
    - The list is crazy long, so I truncated to 10 roles per department... not sure that's right

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/results?core-skills=Scientist%2CDeveloper%2COrganiser%2CAnalyzer%2CManager&filter=home-based&location=africa#available-roles
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Help fix the page...

## Issue / Card

Fixes #353 

## Screenshots
[truncated to 10 roles]

![image](https://user-images.githubusercontent.com/441217/111901865-69070e00-8a32-11eb-81b4-21e0c3a4bddb.png)

[all roles]
![image](https://user-images.githubusercontent.com/441217/111901602-1ed15d00-8a31-11eb-95fd-3f23313bb530.png)
